src/content/zh-cn/updates/2017/09/picture-in-picture.md

```diff
@@ -0,0 +1,98 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: 让我们来讨论如何使用一个Web API以在桌面创建一个浮动视频窗口。
+
+{# wf_updated_on: 2017-09-12 #}
+{# wf_published_on: 2017-09-12 #}
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/picture-in-picture.png #}
+{# wf_featured_snippet: 让我们来讨论如何使用一个Web API以在桌面创建一个浮动视频窗口。 #}
+{# wf_blink_components: Blink>Media #}
+
+# 画中画 (PiP) {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Translated by
```

Comment:
we've typically been asking people to put the translation credit at the bottom of the page.